packages/shared-ui/src/components/Header.tsx

```diff
@@ -298,7 +300,7 @@ function MobileExternalLink({
   )
 }
 
-function HeaderMobile() {
+function HeaderMobile(props: Partial<HeaderProps>) {
```

Comment:
We can be more precise with https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys